doc/nrf/libraries/modem/lte_lc.rst

```diff
@@ -184,16 +184,17 @@ To enable modem sleep and TAU pre-warning notifications, enable the following op
 
 For additional configurations related to these features, see the API documentation.
 
-Connection fallback mode
-========================
-It is possible to try to switch between LTE-M and NB-IoT after a certain time period if a connection has not been established.
+Connection fallback
+===================
+It is possible to try to switch between LTE-M and NB-IoT if a connection has not been established.
 This is useful when the connection to either of these networks becomes unavailable.
-You can also configure the switching period between the network modes.
-If a connection cannot be established by using the fallback mode, the library reports an error.
-You can use the following configuration options to configure the connection fallback mode:
 
-* :kconfig:option:`CONFIG_LTE_NETWORK_USE_FALLBACK`
-* :kconfig:option:`CONFIG_LTE_NETWORK_TIMEOUT`
+Use the :kconfig:option:`CONFIG_LTE_NETWORK_MODE_LTE_M_NBIOT` or :kconfig:option:`CONFIG_LTE_NETWORK_MODE_LTE_M_NBIOT_GPS` Kconfig option
+with which the modem will perform a fallback from one system mode to another if a connection to the first one fails.
+
+In addition, you can control the priority between LTE-M and NB-IoT using the :kconfig:option:`CONFIG_LTE_MODE_PREFERENCE` Kconfig option.
+
+You can also configure the the overall connection establishment time with the :kconfig:option:`CONFIG_LTE_NETWORK_TIMEOUT` Kconfig option.
```

Comment:
```suggestion
You can also configure the overall connection establishment time with the :kconfig:option:`CONFIG_LTE_NETWORK_TIMEOUT` Kconfig option.
```

Reply:
This is now removed after discussing with my colleagues